bin/run.js

```diff
@@ -28,67 +28,76 @@
  * ```
  */
 
-const chalk = require('chalk');
-const yaml = require('js-yaml');
-const fs = require('fs');
-const path = require('path');
-const { spawn } = require('child_process');
-const workspace = process.cwd();
-const { runUses } = require('../lib/uses');
-
-const gitChangedFiles = require('actions-utils/git-changed-files');
-const getBaseRef = require('actions-utils/get-base-ref');
+const chalk = require("chalk");
+const yaml = require("js-yaml");
+const fs = require("fs");
+const path = require("path");
+const { spawn } = require("child_process");
+
+const { runUses } = require("../lib/uses");
+
+const gitChangedFiles = require("actions-utils/git-changed-files");
+const getBaseRef = require("actions-utils/get-base-ref");
+
+// Walk up file system until we find the "workspace root", which must have a `.github/workflow-templates` directory.
+let workspace = process.cwd();
+while (!fs.existsSync(path.join(workspace, ".github/workflow-templates"))) {
```

Comment:
I haven't landed this branch of OLC yet, but this also gets you to the git root: https://github.com/Khan/our-lovely-cli/blob/h11/lib/git-utils.js#L8